pom.xml

```diff
@@ -421,6 +421,35 @@ THE SOFTWARE.
             <maxHeap>768</maxHeap>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-checkstyle-plugin</artifactId>
+          <version>3.1.1</version>
+          <configuration>
+             <consoleOutput>true</consoleOutput>
+             <failsOnError>true</failsOnError>
+             <!-- define here because checkstyle and multi module is a PITA -->
```

Comment:
would be nice to have this in a file, 
here's an example of how we got it working in multi module with jcasc:
https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/pom.xml#L82

non-blocking though

Reply:
Thanks for the pointer. 
Given I have no idea if tools (eclipse intellij netbeans etc) all cope with that correctly (given it is set by the mvn wrapper) - and not everyone imports all the projects in a multi module setup, I'll stick which this for now as I know it to work.